btsieve/src/bitcoin/pattern_matching.rs

```diff
@@ -3,39 +3,51 @@ use bitcoin_support::{
 };
 
 #[derive(Clone, Default, Debug, Eq, PartialEq)]
-pub struct TransactionQuery {
+// If the field is set to Some(foo) then only transactions matching foo are
+// returned Otherwise, when the field is set to None, no pattern matching is
+// done for this field
+pub struct Pattern {
```

Comment:
I like the new name but I think it is too generic. What about `TransactionPattern`?

Reply:
Done, thanks.